AppCenter/AppCenter/Internals/Ingestion/Util/MSTicketCache.h

```diff
@@ -0,0 +1,39 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSTicketCache : NSObject
+
+/**
+ * Dictionary to hold tickets.
+ */
+@property (nonatomic) NSMutableDictionary<NSString *, NSString *> *tickets;
+
+/**
+ * Return singleton instance of MSTicketCache.
+ * @return the instance.
+ */
++ (instancetype)sharedInstance;
+
+/**
+ * Retrieve a ticket from the ticket cache.
```

Comment:
Should be a blank link in between description and `param`s

Reply:
Fixed this!